cc/tflite_micro/tflite_micro.cc

```diff
@@ -71,13 +75,29 @@ int tflite_init(
     return -1;
   }
 
-  // Obtain pointers to the model's input and output tensors.
-  input = interpreter->input(0);
-  output = interpreter->output(0);
+  if (interpreter->inputs_size() > MAX_TENSORS
+    || interpreter->outputs_size() > MAX_TENSORS) {
+    MicroPrintf(
+      "input_tensors: %d, output_tensors: %d, max input/output tensors: %d",
+      interpreter->inputs_size(), interpreter->outputs_size(), MAX_TENSORS);
+    return -1;
+  }
+
+  // Obtain pointers and total size (bytes) of the model's input tensors.
+  for (size_t i = 0; i < interpreter->inputs_size(); i++) {
+    inputs[i] = interpreter->input(i);
```

Comment:
Does this line run the same model multiple times on different inputs?

Reply:
Nope, it prepares all input tensors observed from the model in `inputs[]` which will be used to dispatch input buffer to corresponding input tensors later.